src/api/status/public/js/build-log/datetime-calculator.js

```diff
@@ -0,0 +1,45 @@
+const buildDateCaltulator = document.getElementById('calculatorData');
+const buildLoading = document.getElementById('build-loading');
+const buildSuccess = document.getElementById('build-success');
+const buildFail = document.getElementById('build-fail');
+
+export default function calculatorBuildInfo(data) {
+  const startedAtValue = data.startedAt;
+  const stoppedAtValue = data.stoppedAt;
+  const resultValue = data.result;
```

Comment:
use [destructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here. You can also check why ESLint failed before and the solution for [prefer-destructing](https://eslint.org/docs/rules/prefer-destructuring)